projects/storefrontlib/src/cms-components/configurator/commons/service/config-ui-key-generator.service.unit.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { Configurator } from '@spartacus/core';
+import { ConfigUIKeyGeneratorService } from './config-ui-key-generator.service';
+
+describe('ConfigUIKeyGenerator', () => {
+  let classUnderTest: ConfigUIKeyGeneratorService;
+  const currentAttribute: Configurator.Attribute = {
+    name: 'attributeId',
+    uiType: Configurator.UiType.RADIOBUTTON,
+  };
+
+  beforeEach(() => {
+    classUnderTest = new ConfigUIKeyGeneratorService();
```

Comment:
Bagatelle: Why not to leverage Angular's DI system? 
`classUnderTest = TestBed.get(ConfigUIKeyGeneratorService)`

Reply:
Because it's just a lightweight unit-test. But I've changed it. -> Done